src/processor/processor.cpp

```diff
@@ -86,7 +86,7 @@ void QueryProcessor::decomposePlanIntoTasks(
     } break;
     case PhysicalOperatorType::READER: {
         auto reader = (Reader*)op;
-        if (reader->getContainsSerial()) {
+        if (reader->getContainsSerial() || reader->isCopyTurtleFile()) {
```

Comment:
I thought we only need to read in SERIAL mode rather than copy in single thread

Reply:
We need to execute in single thread mode due to the special copy codepath (append to hashIndex first, then append to columnChunk).